_testnets/namadahousefire/chain.json

```diff
@@ -6,19 +6,12 @@
   "website": "https://namada.net",
   "pretty_name": "Namada",
   "chain_type": "namada",
-  "chain_id": "housefire-below.0b9e6c455af037",
+  "chain_id": "housefire-alpaca.cc0d3e0c033be",
   "slip44": 877,
   "bech32_prefix": "tnam",
   "daemon_name": "unam",
   "key_algos": ["ed25519"],
-  "features": [
```

Comment:
should we enable all features there?

Reply:
This one is set up to progress through phases like mainnet, so we'll start with the current one!